### PR TITLE
Feature/login page layout

### PR DESCRIPTION
**Pull request by kentlouise**
# **Commands**
Frontend
1. Git clone through SSH
2. npm install
3. npm start

# **Pre-conditions**
1. Go to the login page.
2. navigate from home, to login, to register navigation, to check if the page is working.

# **Expected Output**
1. You should be able to see the login page.

# **Notes**
- Login page submit button is not yet function it will be integrated on the next task.

### CODE REVIEW

File: frontend/src/pages/LoginPage.js [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+          aria-label="Toggle navigation">
+          <span className="navbar-toggler-icon"></span>
+        </button>
+        <div className="collapse navbar-collapse" id="navbarNavDropdown">
+          <ul className="navbar-nav">
+            <Navlink navClass="nav-link navLink active font-italic" icon={<i className="bi bi-door-open pr-1"></i>} navLink="/login" navName="Login" />
+            <Navlink navClass="nav-link navLink" icon={<i className="bi bi-person-plus pr-1"></i>} navLink="/register" navName="Register" />
+          </ul>
+        </div>
+      </nav>
```
Comment (kimjo77) [inline frontend/src/pages/LoginPage.js]: I noticed these are being used for `home`, `register`, and `login` page. Usually what we do when components are completely identical we just create a separate component for it so then we can just import and the changes we makes affects all pages. This also help reduce duplication of code.

for this exmaple we'd separate it in another file: `/shared/AuthHeader/AuthHeader.js` (This is just another example you can name it as you feel is proper.

Then store this shareable component there and just import the component on those 3 main pages.

Then I notice you have dynamic active links. react-router-dom have similar dynamic display function so you can use that to make your display for the active route dynamic. is called `<NavLink></NavLink>`. More detailed explanation here for you to experiment.
https://reactrouter.com/web/api/NavLink

File: frontend/src/pages/LoginPage.js [hunk 2]
```diff
@@ -1,7 +1,85 @@
+import Head from '../components/Head';
+import Footer from '../components/Footer';
+import Navlink from '../components/Navlink';
+import FormTemplate from '../components/FormTemplate';
+import { useState, useEffect } from 'react';
+import { Form, Container, Row, Button } from 'react-bootstrap';
+
```
Comment (kimjo77) [inline frontend/src/pages/LoginPage.js]: as general practice we divide different type of imports by space

example
```
// package imports here
import { useState, useEffect } from 'react';
import { Form, Container, Row, Button } from 'react-bootstrap';

// react component imports here
import Head from '../components/Head';
import Footer from '../components/Footer';
import Navlink from '../components/Navlink';
import FormTemplate from '../components/FormTemplate';

// other file imports here
```

please update. no need to include the comments

File: frontend/src/pages/LoginPage.js [hunk 3]
```diff
@@ -1,7 +1,85 @@
...
 export default function LoginPage() {
+
+  const [ email, setEmail ] = useState('');
+  const [ password, setPassword ] = useState('');
+  const [ btnActive, setBtnActive ] = useState(false);
+
+  // control the functionality of submit button
+  useEffect(() => {
+    if((email&& password) !== "") {
```
Comment (kimjo77) [inline frontend/src/pages/LoginPage.js]: add `space` before and `&&`

File: frontend/src/pages/LoginPage.js (hunk 2 above)
Comment (kentlouise) [reply frontend/src/pages/LoginPage.js]: Got it Sir. I'll change it right now.

File: frontend/src/pages/LoginPage.js (hunk 3 above)
Comment (kentlouise) [reply frontend/src/pages/LoginPage.js]: Noted Sir. I'll update this one.

File: frontend/src/pages/RegisterPage.js [hunk 4]
```diff
@@ -1,11 +1,15 @@
+// package imports
```
Comment (kimjo77) [inline frontend/src/pages/RegisterPage.js]: no need these comments haha

Comment (kentlouise) [reply frontend/src/pages/RegisterPage.js]: Ahahaha. Sorry Sir. I'll remove the comments.

**Review by kimjo77 — APPROVED** [review]

**Pull request merged**